packages/palette/src/elements/Button/v3/Button.tsx

```diff
@@ -116,7 +117,7 @@ const Container = styled.button<ContainerProps>`
   ${(props) => {
     // Handle pseudo classes
     return css`
-      @media ${props.theme.mediaQueries.hover} {
+      @media ${MEDIA_QUERY_HOVER} {
```

Comment:
I guess actually I could just import this from each respective theme. I had thought this was weird initially when they were the same component.